### PR TITLE
molmod

### PR DESCRIPTION
**Pull request by jan-janssen**
@conda-forge/help-python-c

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Have a separate PR for MolMod - because of https://github.com/conda-forge/staged-recipes/issues/6167

### CODE REVIEW

File: recipes/molmod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - setuptools
```
Comment (bgruening) [inline recipes/molmod/meta.yaml]: setuptools, if really needed should also be in the host section

**Comment by jan-janssen** [comment]
@bgruening thanks for the review - I included your feedback.

File: recipes/molmod/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/molmod/meta.yaml]: You can remove it from the builds section now.
And maybe you can test it completely without setuptools.

Comment (scopatz) [reply recipes/molmod/meta.yaml]: Please remove setuptools from this section

**Comment by bgruening** [comment]
@jan-janssen thanks for your contribution! Very welcome. Count me in as user :)

**Comment by jan-janssen** [comment]
@bgruening thx, see you next tuesday.

**Pull request merged**